examples/ios/ios/ViewController.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  ViewController.swift
+//  ios
+//
+//
+
+import UIKit
+import Foundation
+import ios_cubic
+
+class ViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSource, UIGestureRecognizerDelegate, CubicManagerDelegate {
+    
+    private static let CUBIC_URL = "demo-cubic.cobaltspeech.com:2727"
+    
+    @IBOutlet weak var modelTextField: UITextField!
+    @IBOutlet weak var resultLabel: UILabel!
+    @IBOutlet weak var recordButton: UIButton!
+   
+    let pickerView = UIPickerView()
+    var models:[Cobaltspeech_Cubic_Model] = []
+    var selectedModel:Cobaltspeech_Cubic_Model?
+    let cubicManager = CubicManager(url: CUBIC_URL)
+        
+    fileprivate func addToolbar() {
+        let toolBar = UIToolbar()
+        toolBar.barStyle = UIBarStyle.default
+        toolBar.isTranslucent = true
+        toolBar.tintColor = UIColor(red: 76/255, green: 217/255, blue: 100/255, alpha: 1)
+        toolBar.sizeToFit()
+        modelTextField.inputAccessoryView = toolBar
+        let doneButton = UIBarButtonItem(title: "Done", style: UIBarButtonItem.Style.done, target: self, action: #selector(self.donePicker))
+        let spaceButton = UIBarButtonItem(barButtonSystemItem: UIBarButtonItem.SystemItem.flexibleSpace, target: nil, action: nil)
+        
+        toolBar.setItems([spaceButton, doneButton], animated: false)
+        toolBar.isUserInteractionEnabled = true
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.cubicManager.delegate = self
+        modelTextField.inputView = pickerView
```

Comment:
While the `UIPickerView` is interesting, I find this interaction with the `modelTextField` a little odd. There is a disconnect between having the model text appear in the `UITextField` at the top of the device, and the `UIPickerView` appear at the bottom (see image below). I think I find this odd for two reasons:

1. A text field implies that the user can type text (and that a keyboard should appear). For this case, we don't want users to type arbitrary text - we want them to select an item from a list.
2. I don't expect to find the list of options in the `UIPickerView` at the bottom of the screen (possibly because I am not a regular iOS user). I would expect it to appear underneath the text field. Especially because there isn't much to this UI, having all that white space in the middle is not very appealing.

Perhaps replacing the text field with a label and a 'Change...' button next to it would help. Or maybe having the `UIPickerView` appear directly below the text field. Or perhaps you can think of something else clever :wink:

<img width="414" alt="Screen Shot 2020-01-16 at 3 48 37 PM" src="https://user-images.githubusercontent.com/26467443/72569453-bc525f00-3877-11ea-84be-91faad0964fe.png">